therr-services/maps-service/src/api/aggregations.ts

```diff
@@ -0,0 +1,31 @@
+const aggregateMetrics = (metrics) => {
+    const formattedMetrics = {};
+
+    metrics.forEach((metric) => {
+        const month = new Date(metric.createdAt).getUTCMonth() + 1;
+        const dayOfMonth = new Date(metric.createdAt).getUTCDate();
+        const dataKey = `${month}/${dayOfMonth}`;
+        formattedMetrics[dataKey] = formattedMetrics[dataKey] !== undefined
+            ? formattedMetrics[dataKey] + Number(metric.value)
+            : Number(metric.value);
+    });
+    return formattedMetrics;
+};
+
+const PercentageChange = (currentSeries, previousSeries) => {
```

Comment:
It's best to be consistent with naming, so could you make this camel cased and a verb (ie. getPerctangeChange)